docs/standard/commandline/2.0.0-beta5.md

```diff
@@ -0,0 +1,330 @@
+---
```

Comment:
I wouldn't name this file the version of the package, a new version means that a redirect has to be created and a file renamed. We also have a standard issue template for a "breaking change" doc, which we can use AI automation to generate details for. This article feels a little more like a "migration from beta 4 to beta 5" and would eventually go away.